lib/defaults/defaults.go

```diff
@@ -693,6 +693,9 @@ const (
 
 	// WebsocketWebauthnChallenge is sending a webauthn challenge.
 	WebsocketWebauthnChallenge = "n"
+
+	// WebsocketSessionData is sending the data for a ssh session.
```

Comment:
Nit: how about we call this `WebsocketSessionMetadata` instead? This way it's clear that it's data _about_ the session, not the actual session data which is `WebsocketRaw`.

Reply:
Good call.